internal/resources/rules/rules.go

```diff
@@ -33,6 +35,6 @@ type RuleSet struct {
 }
 
 type RuleOptions struct {
-	Action     string `yaml:"action,omitempty" json:"action,omitempty"`
-	PrimaryKey string `yaml:"primaryKey,omitempty" json:"primaryKey,omitempty"`
+	Actions    []string `yaml:"actions,omitempty" json:"actions,omitempty"`
```

Comment:
This makes the change in miactl breaking, however you could be using an updated version of miactl with an older console version. Please use the version checking function to discover console version and distinguish which struct should be deserialized